Python/remote_debugging.c

```diff
@@ -422,26 +422,127 @@ search_map_for_section(proc_handle_t *handle, const char* secname, const char* m
 
 #ifdef MS_WINDOWS
 
+static void* _analyze_pe(const wchar_t* mod_path, BYTE* remote_base, const char* substr) {
```

Comment:
No need for the `_` prefix, we didn't do that with any of our other static functions.

Also, I wouldn't call this parameter `substr` since it needs to match only at the start, not at an arbitrary position. It's probably better to use `secname` like the caller does.
```suggestion
static void* analyze_pe(const wchar_t* mod_path, BYTE* remote_base, const char* secname) {
```